uber/site_sections/hotel.py

```diff
@@ -39,6 +39,11 @@ def hours(self, session):
         staffers = [s for s in staffers if s.hotel_shifts_required and s.weighted_hours < 30]
         return {'staffers': staffers}
 
+    def no_shows(self, session):
+        staffers = session.query(Attendee).filter_by(badge_type=STAFF_BADGE).order_by(Attendee.full_name).all()
+        staffers = [s for s in staffers if s.hotel_shifts_required and not s.checked_in]
```

Comment:
``hotel_shifts_required`` isn't the correct option to use here; we should use ``hotel_requests``.  Because currently this won't this won't show staffers who belong to shiftless departments who haven't checked in yet.